boards/arm/nucleo_f746zg/doc/nucleof746zg.rst

```diff
@@ -0,0 +1,189 @@
+.. _nucleo_f746zg_board:
+
+ST Nucleo F746ZG
+################
+
+Overview
+********
+
+The STM32 Nucleo-144 boards provide an affordable and flexible way for users to try out new concepts and build prototypes by choosing from the various combinations of performance and power consumption features, provided by the STM32 microcontroller. For the compatible boards, the SMPS significantly reduces power consumption in Run mode.
+
+The ST Zio connector, which extends the Arduino™ Uno V3 connectivity, and the ST morpho headers provide an easy means of expanding the functionality of the Nucleo open development platform with a wide choice of specialized shields.
```

Comment:
Change to:

    The Arduino-compatible ST Zio connector expands functionality
    of the Nucleo open development platform, with a wide choice
    of specialized Arduino* Uno V3 shields.